test/com/google/javascript/jscomp/PeepholeReplaceArrayOfTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2011 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.javascript.jscomp;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Unit tests for {#link {@link PeepholeReplaceArrayOf}
+ *
+ */
+@RunWith(JUnit4.class)
+public final class PeepholeReplaceArrayOfTest extends CompilerTestCase {
```

Comment:
Consider perhaps a few tests of things that shouldn't change? Array.of.apply, etc? Not sure how realistic those are though.

Reply:
Good idea.